src/Datadog.Trace.ClrProfiler.Managed/Integrations/WebRequestIntegration.cs

```diff
@@ -83,29 +108,50 @@ public static object GetResponse(object webRequest, int opCode, int mdToken, lon
             TargetSignatureTypes = new[] { "System.Threading.Tasks.Task`1<System.Net.WebResponse>" },
             TargetMinimumVersion = Major4,
             TargetMaximumVersion = Major4)]
-        public static object GetResponseAsync(object request, int opCode, int mdToken, long moduleVersionPtr)
+        public static object GetResponseAsync(object webRequest, int opCode, int mdToken, long moduleVersionPtr)
         {
-            return GetResponseAsyncInternal((WebRequest)request);
+            const string methodName = nameof(GetResponseAsync);
+            var webRequestType = webRequest.GetType();
```

Comment:
Looks like we don't need this anymore.

```suggestion
```

Reply:
Thanks for spotting that, I'll remove it. I'm surprised I didn't get a warning or anything about an unused variable.